feather/common/src/entities/player.rs

```diff
@@ -11,6 +11,7 @@ pub fn build_default(builder: &mut EntityBuilder) {
     builder
         .add(Player)
         .add(CreativeFlying(false))
+        .add(PotionEffects(vec![]))
```

Comment:
Do we need this? Is it not better to only add the component when the player actually has potion effects?

Reply:
maybe that will be better?
if so then I'm gonna remove that line